vega_sim/service.py

```diff
@@ -1057,6 +1058,14 @@ def submit_order(
             if price is not None
             else None
         )
+        if round_to_tick and price is not None:
+            # TODO: Implement tick size into the local data cache to avoid API calls
```

Comment:
Could we at least have a dict here to cache it before it's fully integrated into the data cache? I think calling this on every order might slow things down a fair bit. I guess just blindly clear the cache if update market is called in case it's being changed.